README.md

```diff
@@ -1,87 +1,5 @@
-
-```md
-### Team:
-<!--Team name or your github handle if you are a team of one-->
-### Use case:
-<!--A few words about what how your node will be used so we can make recommendations for your setup-->
-
-### Overview:
-<!--How are you running your nodes? What cloud infrastructure? Are you running IPFS out-of-process?-->
-
-*Multiaddress persistence:*
-<!--What are you doing to ensure your multiaddress won't change?-->
-
-*Ceramic State Store persistence:*
-<!--What are you doing to ensure your Ceramic data doesn't get lost?-->
-
-*IPFS Repo persistence:*
-<!--What are you doing to ensure your IPFS data doesn't get lost?-->
-
-*Static IP:*
-<!--Static IP address of the machine your Ceramic daemon runs on.-->
-```
-
-> **Data persistence is a critical step.** Remember that there are no guarantees that other nodes in the network are keeping copies of your streams so you must pin all streams that you care about and back up their data.
->
-> If your nodes restart for any reason and you *do not* have data persistence in place, the following will occur: (1) Your IPFS `config` file will be regenerated with a new multiaddress and you will lose your connection to the rest of the network. (2) The state data for streams pinned on your node will get deleted and likely become unrecoverable. (3) The IPFS data for streams your node has loaded will get deleted and likely become unrecoverable, corrupting your stream state.
->
-> Instructions on configuring your node for proper data persistence can be found in the Ceramic docs on the [Hosting a node](https://developers.ceramic.network/run/nodes/nodes/) page.
-
-Please add one of the maintainers below as a reviewer for your PR. They will do their best to merge PRs as soon as they come in. If you need more immediate assistance, reach out on [Discord](https://chat.ceramic.network) and include the URL of your PR.
-
-## Why peerlist?
-
-Peerlist is a *temporary* solution for peer discovery in Ceramic that will be used up until DAG-JOSE becomes available in go-ipfs. Peerlist is needed because the current JavaScript implementation of the DHT in `libp2p` (a p2p networking library used in Ceramic) is not yet sufficient to deliver optimal performance for a live decentralized network.
-
-## Maintainers
-
-- Val ([@v-stickykeys](https://github.com/v-stickykeys))
-
-## License
-
-Ceramic is fully open source and dual-licensed under MIT and Apache 2.
+# ⛔️ This repo has been superseded by the [Ceramic Anchor Allowlist](https://github.com/3box/ceramic-anchor-allowlist)
+## Want to be added to the Ceramic Anchor Allowlist? 
+Plase visit the [Ceramic Anchor Allowlist Repo](https://github.com/3box/ceramic-anchor-allowlist) and follow the process outlined there! The *Ceramic Anchor Allowlist* gives you access to the 3Box Labs hosted anchor service. The *Peerlist* is no longer being used in the latest Ceramic Versions.
```

Comment:
```suggestion
Please visit the [Ceramic Anchor Allowlist Repo](https://github.com/3box/ceramic-anchor-allowlist) and follow the process outlined there! The *Ceramic Anchor Allowlist* gives you access to the 3Box Labs hosted anchor service.
```

Remove the last line because the peerlist is still being used by Ceramic v2. Or am I wrong here?